Shekinah Glory.onsong

```diff
@@ -3,18 +3,46 @@ Artist: Cory Asbury
 Key: [F]
 Original Key: F
 
-[C]We wait for You, [Fadd9]we wait for You
-[Am]We wait for You; [G]walk in the room
+Verse:
+We wait for [C]You. We wait for [Fadd9]You
+We wait for [Am]You to walk in to [G]room
 
-[C]Here we are, standing in Your presence
-[Fadd9]Here we are, standing in Your presence
-Shekina[Am] glory come down, [G]Shekinah glory come down
+Pre-Chorus:
+[C]Here we are standing in Your presence
+[C]Here we are standing in Your presence
+[F]Shekinah Glory come down. [F]Shekinah Glory come down.
 
-Release the [C]fullness of Your [Fadd9]Spirit
-[Am]Shekinah glory come, [G]Shekinah glory come
+[Am]Here we are standing in Your presence
+[Am]Here we are standing in Your presence
+[G]Shekinah Glory come down. 
 
-[C]You move and we want more, [Fadd9]You speak and we want more
-[Am]You move and we want more; [G]we want the fullness
+Chorus:
+Release the f[C]ullness of Your S[G]pirit
+She[Dm]kinah Glory come. She[F]kinah Glory come
 
-We want more, we want more, we want more, we want more
-We want more, we want more, more of Your Spirit
+Chorus 2:
+You m[C]ove and we want more.
+You sp[G]eak and we want more
+You m[Dm]ove and we want more,
+more of Your Sp[F]irit.
+
+Instrumental:
+C    G    Dm
```

Comment:
These chords won't transpose within the application without being in the [ ] 's. Having them is great but if you transpose the song up a key/two/etc. all the rest of the chords will change and they'll remain the same.